assembl/static2/js/app/utils/videoUtilities.js

```diff
@@ -0,0 +1,54 @@
+// https://en.wikipedia.org/wiki/Video_file_format
+const videoExtensions = [
+  'webm',
+  'mkv',
+  'flv',
+  'vob',
+  'ogv',
+  'ogg',
+  'drc',
+  'gif',
+  'gifv',
+  'mng',
+  'avi',
+  'mov',
+  'qt',
+  'wmv',
+  'yuv',
+  'rm',
+  'rmvb',
+  'asf',
+  'amv',
+  'mp4',
+  'm4p',
+  'm4v',
+  'mpg',
+  'mp2',
+  'mpeg',
+  'mpe',
+  'mpv',
+  'mpg',
+  'mpeg',
+  'm2v',
+  'm4v',
+  'svi',
+  '3gp',
+  '3g2',
+  'mxf',
+  'roq',
+  'nsv',
+  'f4v',
+  'f4p',
+  'f4a',
+  'f4b'
```

Comment:
Okay, I would cut this video into 2 types. VIDEO tag supported types, and OTHER. Here are the video tag supported types: https://developer.mozilla.org/en-US/docs/Web/HTML/Supported_media_formats

You can have `videoExtensions = allVideos + videoTagSupported`. And for EDF, to be accurate, have the utility only look at the videoTagSupported list.

Reply:
not here. It's supposed to be independent of the tag